omnigibson/object_states/open.py

```diff
@@ -76,37 +77,33 @@ def _get_relevant_joints(obj):
             - list of int: Joint directions for each corresponding relevant joint
     """
     global m
-    if not hasattr(obj, "metadata"):
-        return None, None, None
 
-    both_sides = obj.metadata[m.BOTH_SIDES_METADATA_FIELD] if m.BOTH_SIDES_METADATA_FIELD in obj.metadata else False
+    default_both_sides = False
+    default_relevant_joints = list(obj.joints.values())
+    default_joint_directions = [1] * len(default_relevant_joints)
```

Comment:
can we add a comment here? i assume 1 --> open direction corresponds to positive joint angle change, and -1 --> open direction corresponds to negative joint angle change?

Reply:
done